screens/Settings/KeyManagementScreen.tsx

```diff
@@ -0,0 +1,209 @@
+    if (!controller.isKeyManagementTourGuideExplored) {
+      start(t('copilot:keyManagementTitle'));
+    }
+  };
+
+  return (
+    <View
+      style={{
+        flex: 1,
+        elevation: 5,
+        backgroundColor: '#ffffff',
+      }}
+      onLayout={startTourGuide}>
+      <View
+        style={Theme.KeyManagementScreenStyle.outerViewStyle}>
+        <TouchableOpacity onPress={isClosed}>
+          <BackButton onPress={() => {
+            controller.SET_KEY_MANAGEMENT_TOUR_GUIDE_EXPLORED();
+            navigation.goBack()}} />
+        </TouchableOpacity>
+        <Text
+          testID='keyManagementHeadingSettingsScreen'
+          style={Theme.KeyManagementScreenStyle.heading}>
+          {t('header')}
+        </Text>
+        <HelpScreen
+          source={'keyManagement'}
+          triggerComponent={
+            <View testID="help" style={Theme.HelpScreenStyle.viewStyle}>
+              <Row crossAlign="center" style={Theme.HelpScreenStyle.rowStyle}>
+                <View testID="helpIcon" style={Theme.HelpScreenStyle.iconStyle}>
+                  {SvgImage.info()}
+                </View>
+                <Text
+                  testID="helpText"
+                  style={Theme.HelpScreenStyle.labelStyle}>
+                  {t('help')}
+                </Text>
+              </Row>
+            </View>
+          }
+        />
+      </View>
+      <BannerNotificationContainer />
+      <View
+        style={Theme.KeyManagementScreenStyle.copilotViewStyle}>
+        <Copilot
+          title={t('copilot:keyManagementTitle')}
+          description={t('copilot:keyManagementDesc')}
+          order={7}
+          children={
+            <DragList
+              style={Theme.KeyManagementScreenStyle.dragViewStyleSettingsScreen}
+              scrollEnabled={false}
+              data={keyOrder}
+              renderItem={renderItem}
+              keyExtractor={item => item.value}
+              onReordered={handleReorder}
+            />
+          }
+        />
+      </View>
+      <Button
+        testID="saveKeyOrderingPreference"
+        type="gradient"
+        title={t('save')}
+        onPress={async () => {
+          const keyOrderMap = convertToKeyValue(keyOrder);
+          try {
+            controller.SET_KEY_MANAGEMENT_TOUR_GUIDE_EXPLORED();
+            await RNSecureKeystoreModule.storeData(
+              'keyPreference',
+              JSON.stringify(keyOrderMap),
+            );
+            controller.SET_KEY_ORDER_SUCCESS();
+            sendEndEvent(
+              getEndEventData(
+              TelemetryConstants.FlowType.setKeyPriority,
+              TelemetryConstants.EndEventStatus.success)
+            )
+          } catch (e) {
+            sendEndEvent(
+              getEndEventData(
+              TelemetryConstants.FlowType.setKeyPriority,
+              TelemetryConstants.EndEventStatus.failure)
+            )
+            controller.SET_KEY_ORDER_ERROR();
```

Comment:
Instead of handling SET_KEY_ORDER_SUCCESS and SET_KEY_ORDER_ERROR, have single event to handle success and failure response 
controller.SET_KEY_ORDER_RESPONSE(true) or controller.SET_KEY_ORDER_RESPONSE(false).

After this we can remove `isKeyOrderingError` and have only one variable with name `isKeyOrderSet` . 
isKeyOrderSet -> should be true if order is set successfully otherwise false based on the event `SET_KEY_ORDER_RESPONSE`

use `isKeyOrderSet` in Banner to render success or error banner with ternary condition.

Make sure to clean up SET_KEY_ORDER_ERROR, RESET_KEY_ORDER_ERROR